setup.py

```diff
@@ -1,5 +1,31 @@
 from setuptools import setup, find_packages
 
+
+def union(*deps):
```

Comment:
It's obviously not a huge deal, but why define the function here instead of just having it be a one-liner?

Reply:
Just for readability. It's the [Uncle Bob](https://www.amazon.com/Clean-Code-Handbook-Software-Craftsmanship/dp/0132350882) in me. He argues for encapsulating small operations like this in functions and methods to make their purpose a little clearer, even if they're only used once. I think the set union stuff is probably a bit overkill anyway, I think I'm just going to change it to:

```python
extras["dev"] = extras["envs"] + extras["test"] + extras["docs"]
```